packages/fx-core/src/component/generator/generator.ts

```diff
@@ -103,20 +108,28 @@ export class Generator {
     merge(actionContext?.telemetryProps, {
       [TelemetryProperty.SampleName]: sampleName,
     });
+    merge(actionContext?.telemetryProps, {
+      [TelemetryProperty.SampleDownloadDirectory]: isDownloadDirectoryEnabled() ? "true" : "false",
```

Comment:
```
merge(actionContext?.telemetryProps, {
    [TelemetryProperty.SampleName]: sampleName,
    [TelemetryProperty.SampleDownloadDirectory]: isDownloadDirectoryEnabled() ? "true" : "false",
});
```

Reply:
Since the value of isDownloadDirectoryEnabled() is boolean, can we use toString() to convert boolean to string?
```
isDownloadDirectoryEnabled() ? "true" : "false" => isDownloadDirectoryEnabled().toString()
```